std/experimental/allocator/building_blocks/ascending_page_allocator.d

```diff
@@ -212,44 +196,14 @@ public:
     }
 
     /**
-    Returns `true` if the passed buffer is inside the range of virtual adresses.
+    Returns `Ternary.yes` if the passed buffer is inside the range of virtual adresses.
     Does not guarantee that the passed buffer is still valid.
     */
-    bool owns(void[] buf)
+    Ternary owns(void[] buf)
```

Comment:
What's the advantage of using `Ternary` here if there are only two states?

Reply:
I used Ternary for uniformity just as the Region or KRR allocator implement for owns.